core/service/src/engine/threshold/service/key_generator.rs

```diff
@@ -480,6 +446,47 @@ impl<
         Ok(Response::new(Empty {}))
     }
 
+    async fn retrieve_preproc_handle(
```

Comment:
maybe add a comment, that this only retrieves the handle from the store, but does not delete it